superset-frontend/src/components/Icons/AntdEnhanced.tsx

```diff
@@ -17,10 +17,94 @@
  * under the License.
  */
 
-import * as AntdIcons from '@ant-design/icons/lib/icons';
+import {
```

Comment:
would be good to add a comment
```
// Targeted import (as opposed to `import * as antdIcons`) is important here for proper tree-shaking
```

Reply:
ok I added a commit with the comment - first time using that GH feature...

Reply:
> first time using that GH feature...

I'm in love with it... also great for fixing little linting issues on PRs and such.

Reply:
oh nice! never used it myself